nucleus/parent/pom.xml

```diff
@@ -1088,7 +1088,7 @@
                 <plugin>
                     <groupId>org.glassfish.build</groupId>
                     <artifactId>command-security-maven-plugin</artifactId>
-                    <version>1.0.18</version>
+                    <version>1.0.19</version>
```

Comment:
Can 1.0.19 be released to central, please?

Reply:
Yes, I'll do that now. Thanks!